### PR TITLE
cleaning segmentation loss old code

### PR DESCRIPTION
**Pull request by itaijj**

### CODE REVIEW

File: fuse/dl/losses/segmentation/loss_dice.py [hunk 1]
```diff
@@ -44,7 +42,7 @@ def make_one_hot(input: Tensor, num_classes: int) -> Tensor:
 
 
 class BinaryDiceLoss(nn.Module):
-    def __init__(self, power: int = 1, eps: float = 1.0, reduction: str = "mean"):
+    def __init__(self, eps: float = 1e-5, reduction: str = "mean"):
```
[the remark this answers is not in the document]
Comment (itaijj) [reply fuse/dl/losses/segmentation/loss_dice.py:45]: I think that the epsilon for smoothing should be small number and not 1? 
I see in monai the number is 1e-5, do you have any other reference besides @egozi old code to keep it 1??

File: fuse/dl/losses/loss_focalLoss.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        """
+        FL = -alpha(1-pt)^gamma * log(pt)
+        :param alpha: hyperparameter weighted term
+        :param gamma: hyperparameter
+        """
+        super(WeightedFocalLoss, self).__init__()
+        self.alpha = torch.tensor([alpha, 1 - alpha])
+        self.gamma = gamma
+
+    def forward(self, inputs: Tensor, targets: Tensor) -> Tensor:
```
[the remark this answers is not in the document]
Comment (itaijj) [reply fuse/dl/losses/loss_focalLoss.py:57]: waiting to decide if to keep it or not

File: fuse/dl/losses/loss_focalLoss.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    shape[1] = num_classes
+    shape = tuple(shape)
+    result = torch.zeros(shape, device=input.device)
+    result = result.scatter_(1, input, 1)
+
+    return result
+
+
+class WeightedFocalLoss(nn.Module):
+    "Non weighted version of Focal Loss"
```
[the remark this answers is not in the document]
Comment (itaijj) [reply fuse/dl/losses/loss_focalLoss.py:45]: waiting to decide if to keep it or not

**Review by mosheraboh — APPROVED** [review]
